crates/pathfinder/src/sequencer/reply.rs

```diff
@@ -314,11 +314,18 @@ pub mod transaction {
             match self {
                 Transaction::Declare(t) => t.transaction_hash,
                 Transaction::Deploy(t) => t.transaction_hash,
-                Transaction::Invoke(t) => t.transaction_hash,
+                Transaction::Invoke(t) => match t {
+                    InvokeTransaction::V0(t) => t.transaction_hash,
+                    InvokeTransaction::V1(t) => t.transaction_hash,
+                },
             }
         }
     }
 
+    fn default_transaction_version() -> TransactionVersion {
+        TransactionVersion(web3::types::H256::zero())
+    }
+
```

Comment:
Couldn't this be

```
impl Default for TransactionVersion {
  fn default() -> Self { ... }
}
```

Cannot at least see why not ... at least. Could comment if there's a reason?

Reply:
If we can agree on that zero is a meaningful default _everywhere_ then yes, we could totally just do that. I wasn't quite sure that's the case and only thought of this as the default value for deserialization.

Reply:
Went back to the explicit default value only for deserialization -- I couldn't convince myself that the global `Default` makes sense...